static/src/javascripts/polyfill.io

```diff
@@ -0,0 +1 @@
+https://polyfill.guim.co.uk/v2/polyfill.min.js?features=es6,es7,default-3.6&flags=gated&callback=guardianPolyfilled
```

Comment:
with fastly's idiot proof UI, i have managed to do this https://assets.guim.co.uk/polyfill.io/v2/polyfill.min.js

cheers for the prompt 👍